### ISSUE TITLE
Missing docker image for dv_training

### ISSUE THREAD

**Issue by pauline-ng** [Ultimagen/healthomics-workflows#9]
Hi, 

 **TL;DR** Can you add `dv_training` to Docker Hub?
 
 I'm going through the example on the README

`python create_healthomics_workflow.py   single_read_snv   us-east-2   myome-nextflow-workdir   --input_template single_read_snv_template-ppmSeq.json   --omics_workflow_name single_read_snv_wf`

gives an error

`Error response from daemon: pull access denied for ultimagenomics/dv_training, repository does not exist or may require 'docker login': denied: requested access to the resource is denied`

and I notice that [Docker Hub](https://hub.docker.com/search?q=ultimagenomics) does not have the `dv_training` image.

**Comment by pauline-ng** [comment]
Other docker images are missing. If these are set to private , can you set them public?:

```
        "cnv_docker": "ugbio_cnv:1.0.0",
       "dv_training_docker": "ultimagenomics/dv_training:v1.0.7",
       "reads_transformer_docker": "ultimagenomics/ug-reads-transformer:1.0",
        "segdup_docker": "ultimagenomics/parascopy:1.0_30e2e98",
        "vcflite_docker": "ultimagenomics/ugbio_vcflite:1.1.0"
```

**Comment by avigailmo** [comment]
fixed in release: v1.13.4

**Comment by avigailmo** [comment]
all docker issues were fixed in release v1.13.4
